modules/sdk/node-build/esbuild/getExactAliasPlugin.mjs

```diff
@@ -0,0 +1,50 @@
+import path from 'path';
+
+import getImportBridgePath from './getImportBridgePath.mjs';
+
+export default function getExactAliasPlugin(globalImports, moduleName) {
+	const aliases = getAliases(globalImports, moduleName);
+
+	return {
+		name: 'exact-alias-plugin',
+
+		setup(build) {
+			Object.entries(aliases).forEach(
+				([moduleName, filePath]) => {
+					const regexp = new RegExp(`^${moduleName}$`);
+
+					build.onResolve(
+						{
+							filter: regexp
+						}, 
+						args => {
+							return {
+								path: args.path.replace(regexp, path.resolve(filePath))
+							};
+						}
+					);
+				}
+			);
+		},
+	};
+}
+
+function getAliases(globalImports, moduleName) {
+	return Object.entries(globalImports).reduce(
+		(aliases, [globalModuleName, {external, webContextPath}]) => {
+			if (moduleName.endsWith('.css') || globalModuleName === moduleName) {
+				return aliases;
+			}
+
+			if (external) {
+				aliases[globalModuleName] = getImportBridgePath(globalModuleName);
+			}
+			else {
+				aliases[globalModuleName] = `../../../${webContextPath}/__liferay__/exports/index.js`;
```

Comment:
Can we document why we use three steps back in a relative path? Otherwise I'm gonna read this in 3 months and have no idea where it's pointing to

Reply:
just a js comment